brpc-java-core/src/main/java/com/baidu/brpc/client/RpcClient.java

```diff
@@ -107,6 +118,18 @@
      */
     private FastFutureStore fastFutureStore;
 
+    /**
+     * @param service
+     */
+    public void registerPushService(Object service) {
+        ServiceManager.getInstance().registerPushService(service);
+        // 如果只注册了pushService，没有注册一个普通的服务的话， client不会主动和server建立连接， 这里主动进行连接
+        if (instanceProcessor.getInstances().size() == 0) {
+            List<ServiceInstance> lookup = namingService.lookup(new SubscribeInfo());
```

Comment:
here we can not get subscribe info, so remove the lookup service instance logic. Currently we make a convention: push service must be coexist with the normal rpc service, and  RpcClient register push service must be after the RpcClient.getProxy.